misc/docs/source/tutorial/code_snippets/stationxml_custom_tags.rst

```diff
@@ -0,0 +1,167 @@
+.. _stationxml-extra:
+
+=============================================================================
+Handling custom defined tags in StationXML with the Obspy Inventory
+=============================================================================
+
+StationXML allows use of custom elements in addition to the 'usual' information
+defined by the StationXML standard. It allows *a)* custom namespace attributes to
+StationXML namespace tags and *b)* custom namespace subtags to StationXML namespace
+elements.
+ObsPy can handle both basic custom tags in the Inventory object's Network, Station, and Channel objects (*a*) and custom
```

Comment:
I think supporting custom tags on all the main elements is more than sufficient. Anything else is really overkill.